src/borg/remote.py

```diff
@@ -306,6 +316,26 @@ def ssh_cmd(self, location):
             args.append('%s' % location.host)
         return args
 
+    def write_bandwith_limited(self, to_send):
+        if self.ratelimit:
+            current = time.monotonic()
+            if self.ratelimit_last + RATELIMIT_PERIOD <= current:
+                self.ratelimit_quota += self.ratelimit
+                if self.ratelimit_quota > 2 * self.ratelimit:
+                    self.ratelimit_quota = 2 * self.ratelimit
+                self.ratelimit_last = current
+            if self.ratelimit_quota == 0:
+                tosleep = self.ratelimit_last + RATELIMIT_PERIOD - current
```

Comment:
No, in that case `self.ratelimit_last + RATELIMIT_PERIOD <= current` (the if above) and it sets `self.ratelimit_quota += self.ratelimit` (or something close) so `self.ratelimit_quota == 0` will not be true

Or am i confused somehow?